docs/topics/api/addons.rst

```diff
@@ -175,6 +175,9 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json string previews[].image_url: The URL (including a cachebusting query string) to the preview image.
     :>json int previews[].thumbnail_size[]: width, height dimensions of of the preview image thumbnail.
     :>json string previews[].thumbnail_url: The URL (including a cachebusting query string) to the preview image thumbnail.
+    :>json object|null promoted: Object holding promotion information about the add-on. Null if the add-on is not currently promoted.
+    :>json string promoted.category: The name of the promoted category for the add-on.
```

Comment:
I wonder if we should list the possible values of `category` in a table like we do with the other constants the API returns (status, app, platform, etc)